Tasks/ApiCompat/source/task.json

```diff
@@ -1,8 +1,8 @@
 {
-    "id": "7e15a9bc-91c1-488a-b10e-a8106c6e7523",
-    "name": "binaries-comparer",
-    "friendlyName": "Binaries Comparer",
-    "description": "Binaries Comparer",
+    "id": "fd223c26-a98b-4c0d-976f-99086d7e5ae4",
+    "name": "binaries-comparer - Test Wrapping",
```

Comment:
I don't think the `- Test Wrapping` suffix goes here. However, I'd like to rethink the Task name to contain the name of the tool `ApiCompat` or `API Compatibility Verification Tool`.
Let's first figure out were these 3 properties are used in the Marketplace.

Reply:
@JuanAr I reverted the name of the task, since it shouldn't have been committed on these changes. 
The `id` field is used to identify the task, it is a unique identifier.
The `name` field is used to identify the private task.
The `friendlyName` is the name of the task as appears in the marketplace.

For the new name, I would suggest a new PR with more details.

Reply:
Totally agree! New PR.